solr/core/src/test/org/apache/solr/cloud/api/collections/CollectionReloadTest.java

```diff
@@ -86,4 +96,107 @@ public void testReloadedLeaderStateAfterZkSessionLoss() throws Exception {
 
     log.info("testReloadedLeaderStateAfterZkSessionLoss succeeded ... shutting down now!");
   }
+
+  @Repeat(iterations = 5)
```

Comment:
is it worth having this set to 5? how long does this take? should we have maybe 2 here, and set it higher in nightly? not sure if that's possible